static/js/message-translations.ts

```diff
@@ -77,12 +77,12 @@ export const TRANSLATIONS: Record<string, Record<MessageKey, string>> = {
     "Transpile_success": "Good job!\nAmazing!\nWell done!\nExcellent!\nYou did great!",
     "Transpile_warning": "Warning!",
     "Unsaved_Changes": "You have an unsaved program. Do you want to leave without saving it?",
-    "adventures_restored": "The default adventures have been restored!",
+    "adventures_restored": "adventures_restored",
```

Comment:
This also happens to me every time I generate the Typescript, is there a way to prevent this file from being changed? Or should we add such an option?